Users/Models/Users.cs

```diff
@@ -0,0 +1,40 @@
+using System.ComponentModel.DataAnnotations.Schema;
+using Microsoft.Build.Framework;
+using UniVerServer.Abstractions;
+using UniVerServer.Models;
+
+namespace UniVerServer.Users.Models;
+
+[Table("Users")]
```

Comment:
Table annotation not necessary but I could be wrong. It reads your class name in the DB context as the table name